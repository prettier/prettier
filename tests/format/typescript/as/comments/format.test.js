runFormatTest(import.meta, ["typescript", "flow"], {
  errors: { hermes: ["18160.ts"] },
});
