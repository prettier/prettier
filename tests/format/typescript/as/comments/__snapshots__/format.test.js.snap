// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`18160.ts [hermes] format 1`] = `
"';' expected (7:2)
   5 | comment
   6 | */Foo;
>  7 | 1 satisfies /*
     |  ^
   8 | comment
   9 | */Foo;
  10 |
Cause: ';' expected (7:2)
1 satisfies /*
  ^"
`;

exports[`18160.ts format 1`] = `
====================================options=====================================
parsers: ["typescript", "flow"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
1 as /*
comment
*/const;
1 as /*
comment
*/Foo;
1 satisfies /*
comment
*/Foo;

1 as
/*
comment
*/const;
1 as
/*
comment
*/Foo;
1 satisfies
/*
comment
*/Foo;

1 as /*
comment
*/
const;
1 as /*
comment
*/
Foo;
1 satisfies /*
comment
*/
Foo;

1 as
/*
comment
*/
const;
1 as
/*
comment
*/
Foo;
1 satisfies
/*
comment
*/
Foo;

1 as // comment
const;
1 as // comment
Foo;
1 satisfies // comment
Foo;

=====================================output=====================================
1 as const /*
comment
*/;
1 as Foo /*
comment
*/;
1 satisfies Foo /*
comment
*/;

1 as const;
/*
comment
*/
1 as Foo;
/*
comment
*/
1 satisfies Foo;
/*
comment
*/

1 as const /*
comment
*/;
1 as Foo /*
comment
*/;
1 satisfies Foo /*
comment
*/;

1 as const;
/*
comment
*/
1 as Foo;
/*
comment
*/
1 satisfies Foo;
/*
comment
*/

1 as const; // comment
1 as Foo; // comment
1 satisfies Foo; // comment

================================================================================
`;
